pkg/apiserver/resthandler.go

```diff
@@ -308,13 +305,9 @@ func (h *RESTHandler) handleRESTStorage(parts []string, req *http.Request, w htt
 }
 
 // createOperation creates an operation to process a channel response.
-func (h *RESTHandler) createOperation(out <-chan RESTResult, sync bool, timeout time.Duration, onReceive func(RESTResult)) *Operation {
+func (h *RESTHandler) createOperation(out <-chan RESTResult, timeout time.Duration, onReceive func(RESTResult)) *Operation {
 	op := h.ops.NewOperation(out, onReceive)
-	if sync {
-		op.WaitFor(timeout)
-	} else if h.asyncOpWait != 0 {
-		op.WaitFor(h.asyncOpWait)
-	}
+	op.WaitFor(timeout)
```

Comment:
That's fine - I just wasn't sure what the plan was.

----- Original Message -----
> >  	op := h.ops.NewOperation(out, onReceive)
> > -	if sync {
> > -		op.WaitFor(timeout)
> > -	} else if h.asyncOpWait != 0 {
> > -		op.WaitFor(h.asyncOpWait)
> > -	}
> > +	op.WaitFor(timeout)
> 
> I am trying to break this into multiple small manageable changes.
> This PR is to make all operations synchronous and hence never return
> operations to users.
> Then, in next PR, we can delete the /operations endpoint.
> Once these 2 are submitted, we can delete operations all together in a final
> PR.
> 
> Does that sound good or do you think its better to do it all at once?
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/GoogleCloudPlatform/kubernetes/pull/3707/files#r23631838